storage-account-staging.tf

```diff
@@ -18,7 +18,7 @@ resource "azurerm_storage_account" "storage_account_staging" {
 
   location                 = "${azurerm_resource_group.rg.location}"
   account_tier             = "Standard"
-  account_replication_type = "LRS"
+  account_replication_type = "ZRS"
```

Comment:
we have another blob storage account reform-scan
https://github.com/hmcts/reform-scan-shared-infra/blob/master/blob-storage.tf
does it need to be upgraded ?

Reply:
Yes, I am currently working through my list in alphabetical order and will be be submitting a draft PR for that repo as well.